roboslack-api/src/main/java/com/palantir/roboslack/api/attachments/Attachment.java

```diff
@@ -72,7 +72,6 @@ public static Builder builder() {
     @Value.Check
     protected final void check() {
         checkArgument(!Strings.isNullOrEmpty(fallback()), "Attachment fallback message cannot be null or empty");
-        MorePreconditions.checkDoesNotContainMarkdown(FALLBACK_FIELD, fallback());
```

Comment:
I see what you mean. Technically, we could get really creative with our date fallback messages...